client/verta/verta/endpoint/resources/nvidia_gpu.py

```diff
@@ -0,0 +1,75 @@
+from enum import Enum
+
+
+class NvidiaGPU:
+    """
+    Nvidia GPU resources
+
+    The JSON equivalent for this is:
+
+    .. code-block:: json
+
+        {
+            "resources": {"nvidia_gpu": {"number": 1, "model": "V100"}}
+        }
+
+    Parameters
+    ----------
+    number: int > 0
+        Number of GPUs requested to run this endpoint's model.
+    model: :class:`NvidiaGPUModel`, optional
+        Model of GPU requested to run this endpoint's model. Currently available models are
+        specified in the `NvidiaGPUModel` enum.
+
+    Examples
+    --------
+    .. code-block:: python
+
+        from verta.endpoint.resources import Resources, NvidiaGPU, NvidiaGPUModel
```

Comment:
It looks like we're doing 

```suggestion
        from verta.endpoint.resources import Resources
        from verta.endpoint.resources.nvidia_gpu import NvidiaGPU, NvidiaGPUModel
```

I [was deliberately recommending](https://github.com/VertaAI/modeldb/pull/4003#discussion_r1290728666) sticking with having everything `import`able `from verta.endpoint.resources` because I don't think the extra `.nvidia_gpu` is necessary, but I don't mind too much either way (as long as this docstring is correct).